src/utils/TorrClient.ts

```diff
@@ -82,44 +82,34 @@ export const TorrClient = {
   },
 
   resume: async (hash = "") => {
-    return await APICall.get("torrents/resume", {
-      params: {
-        hashes: hash,
-      },
-    });
+    return await APICall.post("torrents/resume", queryString.stringify({
```

Comment:
this will break on browsers because queryString.stringify isn't available. Are you using it through docker?

You could use JSON.stringify which is supported on both node and browsers, however I think it will work without any of that. The second parameter of the `post` method will be the body of the POST

Reply:
yes i am hosting it in docker, using just post will not solve the issue since there is no need for params: object anymore. you are right using JSON maybe be better

Reply:
Uhmm alright

I'd love to keep your contribution in the repo, do you mind reverting back the build folder so I can merge and make the changes needed?